src/Tests/dotnet-workload-update.Tests/GivenDotnetWorkloadUpdate.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine.Parsing;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using FluentAssertions;
+using ManifestReaderTests;
+using Microsoft.DotNet.Cli.NuGetPackageDownloader;
+using Microsoft.DotNet.Workloads.Workload.Install;
+using Microsoft.NET.Sdk.WorkloadManifestReader;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Utilities;
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.DotNet.Workloads.Workload.Install.InstallRecord;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.DotNet.Cli.Workload.Install.Tests;
+using Microsoft.DotNet.Workloads.Workload.Update;
+
+namespace Microsoft.DotNet.Cli.Workload.Update.Tests
+{
+    public class GivenDotnetWorkloadUpdate : SdkTest
+    {
+        private readonly BufferedReporter _reporter;
+        private readonly string _manifestPath;
+        private readonly ParseResult _parseResult;
+
+        public GivenDotnetWorkloadUpdate(ITestOutputHelper log) : base(log)
+        {
+            _reporter = new BufferedReporter();
+            _manifestPath = Path.Combine(_testAssetsManager.GetAndValidateTestProjectDirectory("SampleManifest"), "Sample.json");
+            _parseResult = Parser.GetWorkloadsInstance.Parse(new string[] { "dotnet", "workload", "update" });
+        }
+
+        [Fact]
+        public void GivenWorkloadUpdateItUpdatesOutOfDatePacks()
+        {
+            var mockWorkloadIds = new WorkloadId[] { new WorkloadId("xamarin-android") };
+            (_, var command, var installer, _, _, _) = GetTestInstallers(_parseResult, installedWorkloads: mockWorkloadIds);
```

Comment:
Is there an existing workload installed and then update? Looks like the test setup, there is no existing workload?